anabot/runtime/installation/hub/connect_to_redhat/__init__.py

```diff
@@ -300,16 +327,16 @@ def unregister_button(local_node, visible=True, sensitive=True):
     "Attaching subscription..."
 ]
 registered = [
-        "Registered.",
-        "Registered to Satellite."
+    "Registered.",
```

Comment:
Please don't do those formatting changes as part of one commit (together with implementing the organizations handling). Remove those changes or move them to a separate commit.